ao_mail/models/res_partner.py

```diff
@@ -15,7 +13,7 @@ def _notify(self, message, force_send=False,
         if message.message_type == 'notification':
             notif_partners = self.filtered(
                 lambda partner: partner.mapped(
-                    'user_ids.email_notifications'))
+                    'user_ids.notifications_by_email'))
```

Comment:
-> `notification_by_email`

Reply:
Fixed in https://github.com/Eficent/ao-odoo/pull/59.

Reply:
Solved in other PR. Thanks